### PR TITLE
Refactoring package for Plone 6 

### PR DESCRIPTION
**Pull request by 1letter**
general:
- refactoring package with plone/meta
- use pip as base install method, not buildout
- drop Plone 5 support

package specific:
- generate qr-code as inline base64 image, googlechart is not needed
- add more tests

fixes and features for:
- #11
- #12 
- #13 

- Remove qr_code field from TokenFormAdd, use a Template with a generated Image and a token field. see: https://github.com/collective/pas.plugins.tfa/pull/10/commits/a1aabc099c5f08b4af89b9ade5e72e3deca3f7e1
- fix UserPatch Service Endpoint. see: https://github.com/collective/pas.plugins.tfa/pull/10/commits/0eca90cd2e11df0c9bceb2964f9213fa4896d898
- fix authenticateCredentials in plugin, see: https://github.com/collective/pas.plugins.tfa/pull/10/commits/fa4bd8387426b2bbbb14a95905333ea40c2b4718

### CODE REVIEW

**Review by jensens — COMMENTED** [review]
A huge one. I can not review in detail, but overall LGTM. We may want a second review, like by @mamico

File: setup.py
Comment (mamico) [inline setup.py]:
```suggestion
```
`iw.rejectanonymous` is a soft dependency. My suggestion is to remove it from setup.py and ignore it in the dependency checker.

```
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -121,7 +121,7 @@ Zope = [
   'Products.CMFCore', 'Products.CMFDynamicViewFTI',
 ]
 python-dateutil = ['dateutil']
-
+ignore-packages = ['iw.rejectanonymous']

```

**Review by mamico — APPROVED** [review]
LGTM. Thank you for the great improvement. I only made a small suggestion about a dependency.

File: setup.py
Comment (1letter) [reply setup.py]: @mamico i have took into account your suggestion. i have enable a codecov workflow with upload the summary to codecov.io. can you generate a token on codecov.io and add it to the settings of this package please?

Comment (mamico) [reply setup.py]: I'm sorry, but I don't have access to the settings tab in this collective repository. @jensens, I see you are an owner in the collective organization. Could you possibly help us?

Comment (jensens) [reply setup.py]: I can add a token, but I have no idea what to do at codecov side.

Comment (1letter) [reply setup.py]: @jensens 
> I can add a token, but I have no idea what to do at codecov side.

Nothing is to do, only the token is required to upload the coverage report via gh workflow.

Comment (mamico) [reply setup.py]: @jensens AFAIK you need to open https://codecov.io authenticate using github, search for this repository, or you can probably go directly here: https://app.codecov.io/gh/collective/pas.plugins.tfa/new, now you should see the token for github actions. 
(p.s. I've never used codecov before, so if I'm saying something wrong in the instructions above, please @1letter correct me)

Comment (jensens) [reply setup.py]: `CODECOV_TOKEN` is set. Please try it.

**Pull request merged**
